File: 数组与矩阵/二维数组的查找/solve.java
public class Solution {
    public boolean Find(int target, int [][] array) {
        int row = array.length;
        int col = array[0].length;
        if(row == 1 && col == 0) return false;
        int i = 0; 
        int j = col -1;
        while(array[i][j] != target)
        {
            if(array[i][j] < target)    i++;
            else    j--;
            if(j<0 || i==row) return false;
        }
        return true;
    }
    }

File: 数组与矩阵/二维数组的查找/solve.cpp
class Solution {
public:
    bool Find(int target, vector<vector<int> > array) {
        int row = array.size();
        int col = array[0].size();
        int j = col -1;
        int i = 0;
        for(;j >= 0 && i !=row;) {
            if(target > array[i][j]) i++;
            else if (target < array[i][j]) j--;
            else return true;
        }
        return false;
    }
};
